lib/provider.coffee

```diff
@@ -24,16 +24,17 @@ module.exports =
     scopes = request.scopeDescriptor.getScopesArray()
     isSass = hasScope(scopes, 'source.sass')
 
-    if @isCompletingValue(request)
-      completions = @getPropertyValueCompletions(request)
-    else if @isCompletingPseudoSelector(request)
-      completions = @getPseudoSelectorCompletions(request)
-    else
-      if isSass and @isCompletingNameOrTag(request)
-        completions = @getPropertyNameCompletions(request)
-          .concat(@getTagCompletions(request))
-      else if not isSass and @isCompletingName(request)
-        completions = @getPropertyNameCompletions(request)
+    if not @isCompletingCustomProperty(request)
```

Comment:
Apologies! This is my first shot at coffeescript. It's a frightening departure from Javascript :laughing: